src/Collection.ts

```diff
@@ -356,15 +345,27 @@ export default abstract class Collection<T extends Model> extends Base {
    * use the options to disable adding, changing
    * or removing.
    */
-  fetch({ data, ...otherOptions }: SetOptions = {}): Request {
-    const { abort, promise } = apiClient().get(this.url(), data, otherOptions)
```

Comment:
we lose abort capabilities (which people were not using unluckily), although this has changed a lot nowadays with the introduction to https://developer.mozilla.org/en-US/docs/Web/API/AbortController